config/config_test.go

```diff
@@ -35,3 +42,56 @@ func (s *configSuite) TestPrefixed(c *C) {
 	c.Assert(s.tlc.policy("quux"), Equals, s.tlc.prefixed(rootPolicy, "quux"))
 	c.Assert(s.tlc.volume("foo", "bar", "quux"), Equals, s.tlc.prefixed(rootVolume, "foo", "bar", "quux"))
 }
+
+func (s *configSuite) TestDumpTarball(c *C) {
+	key := "/volplugin/foo"
+	value := "baz"
+
+	exec.Command("/bin/sh", "-c", "etcdctl set "+key+" "+value).Run()
```

Comment:
Thanks for the comment.  Do you really think this is worthwhile?  The code to retrieve the keyspace from etcd is just a [single function call + error check][1], so it doesn't seem like it would really be beneficial to me at this point.

[1]: https://github.com/contiv/volplugin/pull/273/files#diff-3baf47c64847a8fb8aaa8cc2e088513bR136